lib/MusicBrainz/Server/Controller/CDTOC.pm

```diff
@@ -57,9 +57,17 @@ sub show : Chained('load') PathPart('')
     my ($self, $c) = @_;
 
     my $cdtoc = $c->stash->{cdtoc};
+    my $medium_cdtocs = $self->_load_releases($c, $cdtoc);
+
+    for my $mc (@$medium_cdtocs) {
+        ${$mc}{perfect_match} = $c->model('Medium')->perfect_match_cdtoc(
+            ${$mc}{medium}{id},
```

Comment:
It took me a second to figure out this notation. :) It's a Moose object, right? So `$mc->medium->id` should work. Or at least `$mc->{medium}{id}`.

Reply:
notation changed in a0c35bd4444a00e3788a1ceccf2439250cfc138c